server/src/main/java/org/opentosca/toscana/plugins/kubernetes/docker/dockerfile/builder/DockerfileBuilder.java

```diff
@@ -189,6 +212,16 @@ private void writeTo(Writer out) {
         }
     }
 
+    /**
+     Add all EnvCommands at the beginning of the entries list
+     */
+    private void addEnvCommandsToEntries() {
+        for (EnvCommand envCommand : envCommands) {
```

Comment:
you could also use [`entries.addAll(1, envCommands)`](https://docs.oracle.com/javase/8/docs/api/java/util/List.html#addAll-int-java.util.Collection-)